### PR TITLE
added is-cod.in 

### PR DESCRIPTION
**Pull request by Araan-Sheikh**
# Public Suffix List (PSL) Submission

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (`make test`)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the `_psl` TXT record in place in the respective zone(s).

__Submitter affirms the following:__ 

 * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
 

 * [x] This request was _not_ submitted with the objective of working around other third-party limits.

 * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, and responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

 * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms to them.
 * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting.

---

For PRIVATE section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*. I could break my organization's website cookies and cause other issues, and the rollback timing is acceptable. *Proceed anyways*.
---

As you complete each item in the checklist please mark it with an X.
* [x] Report abuse 

[araan@pro.space](mailto:araan@pro.space)

## Description of Organization

Is-cod.in is a subdomain service for people who like coding.

**Organization Website:**

 [is-cod.in](https://is-cod.in)

## Reason for PSL Inclusion
Make our subdomain offering more secure - prevent super cookies and ensuring sites on subdomains stay separate.

**Number of users this request is being made to serve:**

Number of users this request is being made to serve: 80

But we are targeting 2000 users and once we reach the limit we are gonna introduce a new domain.

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -15755,4 +15755,8 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// Company : https://is-cod.in
```
Comment (wdhdev) [inline public_suffix_list.dat:15758]: Please use an actual company name such as:
```suggestion
// is-cod.in : https://is-cod.in
```

**Review by wdhdev — COMMENTED** [review]
1. You do not have 2,000 users, according to https://github.com/is-cod-in/register/tree/main/records you have less than 5, please correct me if I'm wrong however.
2. Please fix your sorting.
3. You are missing your abuse contact.
4. Please provide a non-AI generated organisation description. It's not a marketing statement.
5. You are missing your DNS verification.
6. You have not indicated if you have run tests or not.

It is highly unlikely this will be approved unless you can rectify these issues.

**Comment by Araan-Sheikh** [comment]
> 1. You do not have 2,000 users, according to https://github.com/is-cod-in/register/tree/main/records you have less than 5, please correct me if I'm wrong however.
> 2. Please fix your sorting.
> 3. You are missing your abuse contact.
> 4. Please provide a non-AI generated organisation description. It's not a marketing statement.
> 5. You are missing your DNS verification.
> 6. You have not indicated if you have run tests or not.
> 
> It is highly unlikely this will be approved unless you can rectify these issues.

>we even accept dns request through Discord Bot and yeah we don't have 2000 users, but we are targeting 2k users. Right now we have 100+ users.

**Comment by wdhdev** [comment]
"Targetting" 2,000 users != 2,000 users.

Please note our guidelines below. We tend to require 1,000+ users in order for your request to be approved.

> Projects that are smaller in scale or are temporary or seasonal in nature will likely be declined. Examples of this might be private-use, sandbox, test, lab, beta, or other exploratory nature changes or requets. It should be expected that despite whatever site or service referred a requestor to seek addition of their domain(s) to the list, projects not serving more then thousands of users are quite likely to be declined.

Also, please provide an abuse contact, I noticed you removed that section from your PR template. You cannot just remove a section of the template because you don't want to complete it.

File: public_suffix_list.dat [hunk 2]
```diff
@@ -15755,4 +15755,8 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// is-cod.in : https://is-cod.in
+// Submitted by Araan Sheikh <dotiscodin@gmail.com>
+is-cod.in
+
```
Comment (wdhdev) [inline public_suffix_list.dat:15761]: Fix your sorting please. I would recommend reading the guidelines as it is evident you have not.

Comment (Araan-Sheikh) [reply public_suffix_list.dat:15761]: okay, no time to read each and every sentences lol

Comment (wdhdev) [reply public_suffix_list.dat:15761]: If you aren't able to follow the guidelines and rectify the issues I've listed, you won't be able to be approved. If you don't have time to follow and read the guidelines, I would recommend closing this PR as it won't be approved.

Comment (Araan-Sheikh) [reply public_suffix_list.dat:15761]: Ig it's better to close the pull request right now, will be back when we have 1000+ users.

**Comment by groundcat** [comment]
Unfortunately, the user count does not meet the requirement. We usually ask requestors to ensure they have enough users before submitting to the PSL. Additionally, due to multiple other instances of non-compliance in your PR, it is unlikely to be merged at this time. I recommend closing this PR, and once you have enough users for your service and have thoroughly read and understood the guidelines, you can submit a new PR. All submissions must conform to the [validation and acceptance factors](https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors) and provide sufficient rationale or basically be as complete as possible, and follow the [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines),

**Comment by Araan-Sheikh** [comment]
Closing the PR

**Pull request closed without merging**